src/environments/environment.prod.ts

```diff
@@ -1,6 +1,7 @@
 export const environment = {
-  name: 'prod',
   production: true,
-  APIServerUri: 'http://localhost:3005',
-  APITestResultServerUri: 'http://localhost:3006'
+  VTM_CLIENT_ID: '',
```

Comment:
Should these values not be parameterised?

Reply:
Sorry @premghinde, can you elaborate please?

Reply:
Is the client id likely to be an empty string in production?

Reply:
I see, thank you - no, the build pipeline substitutes this file with one containing production values, this exists in the repo for testing production builds locally

Reply:
Got it. I wasn't aware of that step.